tailwind.config.js

```diff
@@ -17,9 +17,11 @@ module.exports = {
         card: '31.75rem',
         'mobile-card': '41rem',
         'badges-list': '21.25rem',
+        'app-card': '77.75rem',
       },
       maxWidth: {
         100: '6.25rem',
+        1244: '77.75rem',
```

Comment:
what is 1244?

Reply:
1244 is the max-size for both of the appScreen cards outlined here - https://www.figma.com/file/mX1w2b1WxPn7QJ851bcy59/SealCred-May-2022?node-id=2066%3A46357

I originally had this maxWidth named as `big-cards` but Nikita said that it was a confusing name, maybe I should name it to something like `app-content`?

Reply:
Changed it to `app-content`